src/main.c

```diff
@@ -85,27 +85,7 @@ void EnableVCountIntrAtLine150(void);
 
 void AgbMain()
 {
-#if MODERN
```

Comment:
These was a reason this was added you can't just say it isn't needed anymore if one of your modern builds doesn't blow up because of it and oh hey i just read the comment so `RegisterRamReset **may** crash if it resets IWRAM.`

Reply:
Not a single compiler I tried from 9.3 onwards has had this issue. Not one.

Reply:
you can't rule it out

Reply:
what do you know maybe someone using 9.1 will have this issue and we can't do anything about it

Reply:
9.1 Works. Want me to test any other compiler?